python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -79,6 +87,10 @@ def start_ganga(gangadir_for_test='$HOME/gangadir_testing', extra_opts=[]):
         else:
             logger.info("InternalServices still running")
 
+        # The queues are shut down by the atexit handlers so we need to start them here
+        from Ganga.Core.GangaThread.WorkerThreads import startUpQueues
+        startUpQueues()
```

Comment:
Ah, so it does, in that case never mind my comment :)

Once 'ganga' has replicated 90% of the functionality of the bootstrap we can just use that rather than trying to fix too much in the bootstrap now :)